pints/_mcmc/__init__.py

```diff
@@ -293,13 +293,60 @@ class MCMCController(object):
     method : class
         The class of :class:`MCMCSampler` to use. If no method is specified,
         :class:`HaarioBardenetACMC` is used.
+    transform : pints.Transform
```

Comment:
I think I'd prefer the keep method as the final argument

Reply:
Done